client/app/previous-pres/previousPres.controller.js

```diff
@@ -29,9 +31,16 @@ angular.module('spkr.previous-pres', ['ngRoute'])
           feedback.scores.forEach(function(score,i){
             distData[score][i]++;
           });
+          if ( feedback.comments ) {
+            $scope.comments.push(feedback.comments);
+          }
         });
         //call the presentationGraph factory function (this is where d3 happens)
         Vis.presentationGraph(criteria, distData);
+        if ( $scope.comments.length === 0 ) {
```

Comment:
I would prefer that we either comment out or make this code function predictably, to avoid potential problems in the future.